lib/views/movie_deck.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:flutter/material.dart';
+import 'package:the_movie_db/models/repository/genre_repository.dart';
+import 'package:the_movie_db/widget/loader_image.dart';
+import '../widget/genres_row.dart';
+import '../models/genre.dart';
+import '../widget/score_card.dart';
+import '../widget/like_button.dart';
+import '../utils/general_constants.dart';
+import '../models/movie.dart';
+
+class MovieDeck extends StatelessWidget {
+  final Movie movie;
+  static const String genresString = 'Genres';
+  static const String description = 'Description';
+  static const double sizeFontReleaseDate = 13;
+  static const int position = 0;
+
+  const MovieDeck({
+    required this.movie,
+    super.key,
+  });
+
+  List<Genre> initializeData(
+      AsyncSnapshot<List<dynamic>> snapshot,
+      ) {
+    List<dynamic>? data = snapshot.data;
+    List<Genre> genres = data?[position];
+    return genres.where(
+          (genre) {
+        return movie.genres.contains(genre.id);
+      },
+    ).toList();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+      future: Future.wait(
+        [
+          GenreRepository().getData(),
+        ],
+      ),
```

Comment:
You can wait to a single future:
`future: GenreRepository().getData()`.
Also, we can add some best practices here. Please read [this article](https://api.flutter.dev/flutter/widgets/FutureBuilder-class.html#:~:text=a%20Future.-,Managing%20the%20future,-The%20future%20must) to refactor this widget.